src/admin/containers/labels.js

```diff
@@ -33,18 +32,25 @@ const lines =
   , 'address3'
   , 'address4'
   , 'postcode'
-  , 'county'
+  , 'deliverer'
   ]
 
 
 const font_style = text =>
   ({ style: { fontSize: Math.min(12, 300 / text[0].length) + 'pt' } })
 
+
+const add_deliverer = arr =>
```

Comment:
Out of interest @rjmk can you briefly explain why it is better like this?

We were wondering how we could write this without specifying the 'arr', using only ramda functions..  could you suggest something to look into?